src/video/SDL_egl.c

```diff
@@ -537,6 +537,7 @@ bool SDL_EGL_LoadLibrary(SDL_VideoDevice *_this, const char *egl_path, NativeDis
                 }
             }
             _this->egl_data->egl_display = _this->egl_data->eglGetPlatformDisplay(platform, (void *)(uintptr_t)native_display, attribs);
+            SDL_free(attribs);
```

Comment:
Should it be an error to return NULL attribs? We pass NULL to eglGetPlatFormDisplay() if there isn't a callback set...

Reply:
They could just pass an empty array.